src/test/resources/sample_docs/Iso19115/output.json

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
`output.json` seems like an odd name for this file?

You want this to be valid JSON not JSONL, right? so let's do two records?

```
[
  { 
    // record1
   },
  {
    // record2
  }
]
```